src/consensus/aft/test/driver.h

```diff
@@ -85,21 +85,21 @@ struct LoggingStubStoreSig_Mermaid : public aft::LoggingStubStoreSigConfig
                          tx_id.version,
                          t)
                     << std::endl;
-    aft::LoggingStubStoreSigConfig::rollback(tx_id, t);
+    aft::LoggingStubStoreConfig::rollback(tx_id, t);
   }
 
   void initialise_term(aft::Term t) override
   {
     RAFT_DRIVER_OUT << fmt::format(
                          "  {}->>{}: [KV] initialising in term {}", _id, _id, t)
                     << std::endl;
-    aft::LoggingStubStoreSigConfig::initialise_term(t);
+    aft::LoggingStubStoreConfig::initialise_term(t);
   }
 };
 
 using ms = std::chrono::milliseconds;
 using TRaft = aft::Aft<LedgerStubProxy_Mermaid>;
-using Store = LoggingStubStoreSig_Mermaid;
+using Store = LoggingStubStore_Mermaid;
```

Comment:
All of the changes under `test/` are related to this, and aiming to resolve a Trace Validation failure. The driver was using this "everything is a signature" mock of the `kv::Store`, which meant that the backups viewed _all_ entries as committable (while the leader only committed those from `emit_signature`). The change is to move the committability (as encoded in an `ExecutionWrapper::result`, for _reasons_) from a compile-time template (I don't quite remember why that ever made sense) to embedding it in the serialised entry, as we do with the view and term. A related cleanup removes all of the now-unnecessary (and inaccurate) mock types with `Sig` in the name.

Reply:
I think we should make the payloads JSON next, so they can be easily accessed from the trace validation if desired. And also because why not.

Reply:
Agreed - it looks like we already do this for the reconfiguration messages, so assume we can do it more generally.